infra/makemake/buildbot.nix

```
@@ -0,0 +1,36 @@
+{ lib, ... }: {
+  services = {
+    caddy.virtualHosts."buildbot.ngi.nixos.org".extraConfig = ''
+      reverse_proxy localhost:8010 {
+        header_up X-Forwarded-Proto https
+        header_up X-Forwarded-Port 443
+      }
```

```suggestion
      reverse_proxy localhost:8010
```

Are these `X-*` really required by buildbot? I checked in buildbot-nix, where I don't see them:

https://github.com/Mic92/buildbot-nix/blob/077a60a5d040f7161c50a83d827db77a2a68ed9f/nix/master.nix#L368-L383

And from what I know, NGINX also does not send these by default.

---

They're not required by buildbot, but they're not affecting things in a bad way either. I agree however that we should either add the most important ones, i.e. [x-forwarded-for](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) that's useful when looking at logs, or remove all of them.